### DISCUSSION TITLE
New bug upon retrying?

### DISCUSSION THREAD

**Discussion by reissfelt** [ppy/osu#30741] [Q&A] [answered]
Usually when i play and retrying a beatmap via ` key or 'Ctrl + R' shortcut, it'll usually shows up beatmap details, and Play Settings (Audio, Video, and Input Settings), but now it doesn't.

It doesn't show this bug on manual retry through pause menu, but with ` key or 'Ctrl + R' shortcut it wouldn't show anything. Is it a bug just on me?

**Comment by peppy** [comment, marked as answer]
This was an intentional change because people wanted "instant retry". It turns out that not everyone does, such as you.

We'll probably iterate on this and make it possible to access settings again or make it a setting.

**Reply by reissfelt** [reply]
thanks for the consideration!
